github/PullRequest.py

```diff
@@ -727,6 +727,26 @@ def is_merged(self) -> bool:
         status, headers, data = self._requester.requestJson("GET", f"{self.url}/merge")
         return status == 204
 
+    def enable_automerge(self, merge_method: Opt[str] = "MERGE") -> None:
```

Comment:
Should mention allowed values for `merge_method` in doc string: "MERGE", "REBASE", "SQUASH".